src/data/ListData.jsx

```diff
@@ -0,0 +1,44 @@
+const listData = [
```

Comment:
Since you are not returning any jsx from this file, change the file extension to `.js`.